### PR TITLE
Some XLA utility function in hlo_creation_utils

### PR DESCRIPTION
**Pull request by nouiz**
A split from: https://github.com/tensorflow/tensorflow/pull/32251

### CODE REVIEW

File: tensorflow/compiler/xla/service/hlo_creation_utils.h [hunk 1]
```diff
@@ -107,6 +110,14 @@ StatusOr<HloInstruction*> MakeConcatHlo(
 // the given primitive type.
 HloInstruction* MakeConvertToHlo(HloInstruction* hlo, PrimitiveType type);
 
+// Create a BitcastConvert HLO instruction.
+HloInstruction* MakeBitcastConvertToHlo(HloInstruction* hlo,
+                                        PrimitiveType type);
+
+// Creates an Iota HLO instruction
```
Comment (cheshire) [inline tensorflow/compiler/xla/service/hlo_creation_utils.h]: Dot after sentence

File: tensorflow/compiler/xla/service/hlo_creation_utils.h [hunk 2]
```diff
@@ -107,6 +110,14 @@ StatusOr<HloInstruction*> MakeConcatHlo(
 // the given primitive type.
 HloInstruction* MakeConvertToHlo(HloInstruction* hlo, PrimitiveType type);
 
+// Create a BitcastConvert HLO instruction.
```
Comment (cheshire) [inline tensorflow/compiler/xla/service/hlo_creation_utils.h]: Creates

**Review by cheshire — CHANGES_REQUESTED** [review]
Two minor changes, LGTM otherwise.

**Comment by nouiz** [comment]
Fixed.

**Review by cheshire — APPROVED** [review]

**Comment by nouiz** [comment]
Any idea what is going on with the 'Ubuntu CC' CI? It didn't run in 3 days.

**Comment by cheshire** [comment]
This change got imported, I need to make sure it runs locally before it will get committed. I think external bots are fine with this PR.

**Pull request merged**